src/Xamarin.Android.Build.Utilities/Sdks/AndroidSdkWindows.cs

```diff
@@ -15,17 +15,13 @@ class AndroidSdkWindows : AndroidSdkBase
 		const string XAMARIN_ANDROID_INSTALLER_PATH = @"SOFTWARE\Xamarin\MonoAndroid";
 		const string XAMARIN_ANDROID_INSTALLER_KEY = "PrivateAndroidSdkPath";
 
-		public override string Adb { get { return "adb.exe"; } }
-		public override string Android { get { return "android.bat"; } }
-		public override string Emulator { get { return "emulator.exe"; } }
-		public override string Monitor { get { return "monitor.bat"; } }
-		public override string ZipAlign { get { return "zipalign.exe"; } }
-		public override string JarSigner { get { return "jarsigner.exe"; } }
-		public override string KeyTool { get { return "keytool.exe"; } }
-		public override string NdkStack { get { return "ndk-stack.exe"; } }
+		public override string ZipAlign { get; protected set; } = "zipalign.exe";
```

Comment:
One of the great side-effects of the "let's do this for *everything*" approach is that we shouldn't need to override these properties anymore. We thus shrink `AndroidSdkWindows` and `AndroidSdkUnix`.